jobs/geo/tiger.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+from jobs.utils.snowflake import gdf_to_snowflake, snowflake_connection_from_environment
+
+
+def load_geo_data(conn, year: int) -> None:
+    """Load CA Census geo data into Snowflake."""
+    from fiona.errors import DriverError
+    from pygris import (
+        block_groups,
+        blocks,
+        coastline,
+        combined_statistical_areas,
+        core_based_statistical_areas,
+        counties,
+        county_subdivisions,
+        divisions,
+        nation,
+        native_areas,
+        places,
+        primary_roads,
+        primary_secondary_roads,
+        pumas,
+        rails,
+        regions,
+        states,
+        tracts,
+        tribal_block_groups,
+        tribal_subdivisions_national,
+        urban_areas,
+    )
+
+    print(f"Downloading data for CA in year {year}")
+
+    ca_loaders = {
+        f"COUNTIES_{year}": counties,
+        f"TRACTS_{year}": tracts,
+        f"BLOCK_GROUPS_{year}": block_groups,
+        f"BLOCKS_{year}": blocks,
+        f"PLACES_{year}": places,
+        f"PUMAS_{year}": pumas,
+        f"COUNTY_SUBDIVISIONS_{year}": county_subdivisions,
+        f"PRIMARY_SECONDARY_ROADS_{year}": primary_secondary_roads,
```

Comment:
I'm open to pushback on this, but I think it might be slightly better to not have the year in the table name, and just have it in the schema name. That would make it easier to write scripts/models that just reference, e.g., `COUNTIES`, and select different years by using the appropriate schema. So there would be less work to do to switch between different years if necessary.

Reply:
good point, agreed!